### ISSUE TITLE
Update Notifications

### ISSUE THREAD

**Issue by GetCre8ive** [PoC-Consortium/Qbundle#69]
Can you change this so updates only show up for applications I have installed?

**Comment by HeosSacer** [comment]
Is the "NEW UPDATES AVAILABLE"-thing still showing up after the 2.5.0 upgrade?

**Comment by GuyCre8ive** [comment]
Yeah all my wallets show that there's a new update available but everything I have installed is up to date.

**Comment by HeosSacer** [comment]
Could you provide a screenshot with your Qbundle-main-window and your update-manager screen?

**Comment by GuyCre8ive** [comment]
![qbundle](https://user-images.githubusercontent.com/6698995/43685700-e2f99c2e-9885-11e8-90d7-b075a50a994d.jpg)

**Comment by HeosSacer** [comment]
Hello @GetCre8ive, could I get a listing of files you have inside your qbundle dir? The strange thing is, your blage miner lacks a installed version, despite it is running. Do you have a Qbundle/BlagoMiner/version file?
